src/models/HTTPResponse.ts

```diff
@@ -0,0 +1,25 @@
+import {APIGatewayProxyResult} from "aws-lambda";
+export class HTTPResponse extends Error implements APIGatewayProxyResult {
```

Comment:
Should this be an error? I feel like that would have bad consequences. May have been an error in my 3330 code

Reply:
hmmm... test number does this. Still feels wrong.

Reply:
good point ... I've made the changes